civis/tests/test_io.py

```diff
@@ -107,11 +107,10 @@ def test_zip_member_to_civis(self, *mocks):
 
     @mock.patch(api_import_str, return_value=civis_api_spec)
     def test_file_to_civis(self, *mocks):
-        with tempfile.NamedTemporaryFile() as tmp:
-            tmp.write(b'a,b,c\n1,2,3')
-            tmp.flush()
-            tmp.seek(0)
-            result = civis.io.file_to_civis(tmp, tmp.name)
+        buf = StringIO()
```

Comment:
At a glance, it seems like we ought to also have a bytes-version of this in here:

```
buf = BytesIO()
buf.write(b'a,b,c\n1,2,3')
buf.seek(0)
result = civis.io.file_to_civis(buf, 'somename')
assert isinstance(result, int)
```

Reply:
I'll rename this to `test_text_file_to_civis` and add a `test_bytes_file_to_civis`.

Also, for transparency, I discussed these tests with Stephen and we agreed that we should add integration tests similar to what have been implemented in the R client where we actually test different files and validate not only that they were successfully uploaded and but also that they were uploaded with the correct contents. Those tests are not within the scope of this PR though.